standards/SPIFFE_Workload_API.md

```diff
@@ -10,24 +10,24 @@ Portable and interoperable cryptographic identity for networked workloads is per
 
 ## Table of Contents
 
-1\. [Introduction](#1-introduction)
-2\. [Extensibility](#2-extensibility)
-3\. [Service Defintion](#3-service-definition)
-4\. [Client and Server Behavior](#4-client-and-server-behavior)
-4.1. [Identifying the Caller](#41-identifying-the-caller)
-4.2. [Connection Lifetime](#42-connection-lifetime)
-4.3. [Stream Responses](#43-stream-responses)
-4.4. [Default Values and Redacted Information](#44-default-values-and-redacted-information)
-4.5. [Mandatory Fields](#45-mandatory-fields)
-4.6. [Federated Bundles](#46-federated-bundles)
-5\. [X.509-SVID Profile](#5-x509-svid-profile)
-5.1. [Profile Definition](#51-profile-definition)
-5.2. [Profile RPCs](#52-profile-rpcs)
-5.3. [Default Identity](#53-default-identity)
-6\. [JWT-SVID Profile](#6-jwt-svid-profile)
-6.1. [Profile Definition](#61-profile-definition)
-6.2. [Profile RPCs](#62-profile-rpcs)
-6.3. [JWT-SVID Validation](#63-jwt-svid-validation)
+- 1\. [Introduction](#1-introduction)
```

Comment:
These TOC lines were all collapsed and I'm not sure how to get them all on their own line w/o putting them in a bullet list. Happy to have alternate suggestions.

Reply:
Looking at other docs where the TOC is not collapsed, try adding two spaces after each line.

Reply:
Thanks, @spikecurtis. That did the trick.